pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -455,3 +458,49 @@ func aggregatePersistentVolumeClaimStatus(object *unstructured.Unstructured, agg
 	pvc.Status = *newStatus
 	return helper.ToUnstructured(pvc)
 }
+
+func aggregatePodDisruptionBudgetStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
+	pdb := &policyv1.PodDisruptionBudget{}
+	err := helper.ConvertToTypedObject(object, pdb)
+	if err != nil {
+		return nil, err
+	}
+
+	newStatus := &policyv1.PodDisruptionBudgetStatus{
+		DisruptedPods: make(map[string]metav1.Time),
+	}
+	for _, item := range aggregatedStatusItems {
+		if item.Status == nil {
+			continue
+		}
+
+		temp := &policyv1.PodDisruptionBudgetStatus{}
+		if err = json.Unmarshal(item.Status.Raw, temp); err != nil {
+			return nil, err
+		}
+		klog.V(3).Infof(
+			"Grab pdb(%s/%s) status from cluster(%s), desired healthy: %d, current healthy: %d, disrupted: %d, expected: %s",
+			pdb.Namespace, pdb.Name, item.ClusterName,
+			temp.DesiredHealthy, temp.CurrentHealthy, temp.DisruptedPods, temp.ExpectedPods,
+		)
+
+		newStatus.CurrentHealthy += temp.CurrentHealthy
+		newStatus.DesiredHealthy += temp.DesiredHealthy
+		newStatus.ExpectedPods += temp.ExpectedPods
+		newStatus.DisruptionsAllowed += temp.DisruptionsAllowed
+		newStatus.Conditions = append(newStatus.Conditions, temp.Conditions...)
```

Comment:
What if the same `reason` condition occurs in the different clusters?

Reply:
I would want them all still aggregated and combined. Ideally, it would contain the member cluster name but there are no fields to squeeze that in.

Here is an example of a condition struct:
```
  conditions:
  - lastTransitionTime: "2023-01-04T17:17:54Z"
    message: ""
    observedGeneration: 1
    reason: SufficientPods
    status: "True"
    type: DisruptionAllowed
```

Reply:
We don't plan on using the conditions for any monitors or metrics. It can, however, be helpful for live debugging.

I'm not certain what you're asking to change here, can you elaborate?

Reply:
We don't plan on using the conditions for any monitors or metrics. It can, however, be helpful for live debugging.

I'm not certain what you're asking to change here, can you elaborate?

Reply:
I think it's not useful to collect the condition, due to the existence of redundant information.